specification/mzlb_json.schema.json

```
@@ -0,0 +1,145 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

We should also add `$id`:
```suggestion
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "https://raw.githubusercontent.com/HUPO-PSI/mzSpecLib/master/specification/mzlb_json.schema.json",
```